velox/functions/sparksql/Register.cpp

```diff
@@ -213,6 +220,18 @@ void registerFunctions(const std::string& prefix) {
   // Register bloom filter function
   registerFunction<BloomFilterMightContainFunction, bool, Varbinary, int64_t>(
       {prefix + "might_contain"});
+
+  registerArrayMinMaxFunctions<int8_t>(prefix);
+  registerArrayMinMaxFunctions<int16_t>(prefix);
+  registerArrayMinMaxFunctions<int32_t>(prefix);
+  registerArrayMinMaxFunctions<int64_t>(prefix);
+  registerArrayMinMaxFunctions<int128_t>(prefix);
+  registerArrayMinMaxFunctions<float>(prefix);
+  registerArrayMinMaxFunctions<double>(prefix);
+  registerArrayMinMaxFunctions<bool>(prefix);
+  registerArrayMinMaxFunctions<Varchar>(prefix);
+  registerArrayMinMaxFunctions<Timestamp>(prefix);
+  registerArrayMinMaxFunctions<Date>(prefix);
```

Comment:
Do we need Date? It should be int32_t

Reply:
If we remove Date type from substrait in Gluten Transformer, then we can remove Date here.

I can remove Date in this PR, If needed in the future, it can be added back.

Reply:
@aditi-pandit Could you help look into this? Do we still need to register Date even if we have registered with int32_t?

Reply:
@mbasmanova Do we should register for `Date`?

Reply:
I assume we need to register for Date, but you can double check by adding a test that uses Date columns. I expect the test to fail without this registration call. Please, try that out.

Reply:
checked, we should register for Date

Reply:
Thank you for checking.